examples/lighting-app/telink/Readme.md

```diff
@@ -0,0 +1,180 @@
+![Telink B91 EVK](http://wiki.telink-semi.cn/wiki/assets/Hardware/B91_Generic_Starter_Kit_Hardware_Guide/connection_chart.png)
+
+## Build
+1. Pull docker image from repository:
+    ```
+    $ docker pull alexkolosovtelinksemi/chip-build-telink:latest
+    ```
+
+1. Run docker container:
+    ```
+    $ docker run -it -v ${CHIP_BASE}:/root/chip alexkolosovtelinksemi/chip-build-telink:latest
+    ```
+    here `${CHIP_BASE}` is directory which contains CHIP repo files
+    **!!!Pay attention that OUTPUT_DIR should conatins ABSOLUTE path to output dir**
+
+1. Run build script:
+    ```
+    cd /root/chip/scripts/examples && ./telink_example.sh
+    ```
+1. Exit docker container and collect build artifacts. Firmware binary would be located in **${CHIP_BASE}/examples/lighting-app/telink/build/tlsr9518adk80d/zephyr/zephyr.bin**
+
+## Flash
+1. Download toolchain:
```

Comment:
existing scripts create a `*.flash.py` output from platforms so that we can compile in the docker image and then flash from my host. Please confirm that this is possible for telink as well.

These flashing instructions seem quite complex.

Reply:
@andy31415 Thanks for feedback. Currently I did not implement such script. I guess it shouldn't take much time to implement it. Shell it be done in scope of this one PR or it could be done with separate PR?

Reply:
Ideally quick flashing would be part of the first PR, to enforce a consistent experience for developers.

Reply:
@andy31415 Now it is possible to do flashing via **west flash** command. Corresponding section in Readme file has been updated. It still requires some presetup. Does this solution is fine or it is still to complex?